update Makefile for test

wenhuizhang opened the pull request.

wenhuizhang committed:
update Makefile for test

phil-lopreiato commented on server/test/Makefile:
```diff
@@ -1,19 +1,34 @@
+	-./test-ht-wrapper
+	./server-kv-test &
+	sleep 1
+	./client-kv-test
```
[Tests are failing](https://travis-ci.org/gwAdvNet2015/gw-kv-store/builds/60625118#L380) because this isn't compiled anywhere.

phil-lopreiato commented:
@wenhuizhang why do we need this? #73 is taking care of client testing. The executable you're running aren't even compiled anywhere. Closing for now.

wenhuizhang commented:
Oh, in my humble opinion, #73 is taking care of tests for client, yet not
for server, if I did not make it wrong, please correct me if I did.

On Wed, Apr 29, 2015 at 5:51 PM, Phil Lopreiato notifications@github.com
wrote:

> @wenhuizhang https://github.com/wenhuizhang why do we need this? #73
> https://github.com/gwAdvNet2015/gw-kv-store/pull/73 is taking care of
> client testing. The executable you're running aren't even compiled
> anywhere. Closing for now.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gwAdvNet2015/gw-kv-store/pull/74#issuecomment-97622742
> .

## 

V/R,

Wenhui Zhang

Email: wenhui@gwmail.gwu.edu
           Telephone: 1-(703) 424 3193

phil-lopreiato commented:
I think the tests for the server code are in a pretty good state right now, so I don't think any more are necessary. 

Plus, there isn't even any C code referenced by your makefile - read the test output to see where. 

Unless @atryus disagrees, I think this one should stay closed.

amygara commented:
I agree with @phil-lopreiato about keeping this closed. Phil and I will talk tomorrow and figure out what needs to be refactored. We will create an issue tomorrow instructing exactly what needs to be done.

wenhuizhang commented:
Just updated and changed dir for include headers in server-kv-test and client-kv-test, and it works for me now, created a new push, pls let me know if you have any questions @zhangwei1984  @phil-lopreiato @atryus
